lib/twine/formatters/android.rb

```diff
@@ -125,10 +125,13 @@ def key_value_pattern
         "\t<string name=\"%{key}\">%{value}</string>"
       end
 
+      # see http://developer.android.com/guide/topics/resources/string-resource.html#FormattingAndStyling
+      # however
+      #   - escaping double qoutes with HTML entities works even for getResources().getString()
+      #   - unescaped HTML markup like in "Welcome to <b>Android</b>!" is stripped when retrieved with getString()
```

Comment:
@yincrash you mentioned in #35 that those formatting tags should **not** be escaped. Am I missing something?